### PR TITLE
Massa Station Linux HTTPS workaround

### PR DESCRIPTION
**Pull request by eguoli**
This is a common problem for Linux Massa Station users - https protocol is not available by default without adding CA to the browser.

### CODE REVIEW

**Review by mazmassa — APPROVED** [review]

File: INSTALLATION.md [hunk 1]
```diff
@@ -80,6 +80,11 @@ Congratulations! You have successfully installed **Massa Station** on your compu
 5. Proceed to GUI installation starting at step 2. 
 
 
+### Linux HTTPS workaround
```
Comment (Thykof) [inline INSTALLATION.md:83]: I would suggest to change the title:
```suggestion
### Manually install the HTTPS certificate
```

File: INSTALLATION.md [hunk 2]
```diff
@@ -80,6 +80,11 @@ Congratulations! You have successfully installed **Massa Station** on your compu
...
+1. Open your browser "Manage Certificates" settings (e.g. chrome://settings/certificates or brave://settings/certificates) and switch to the "Authorities" tab.
```
Comment (Thykof) [inline INSTALLATION.md:84]: I think we could add a sentence explain why user could need this manual step.

**Review by Thykof — COMMENTED** [review]
Thank you for your contribution! Really appreciated! On Linux, with Chrome browser, latest version of Massa Station is supposed to work with HTTPS without any manual certificate installation. However, this documentation steps are useful for any other browser: explain to user that it requires to manually install the certificate onto the browser.

File: INSTALLATION.md (hunk 2 above)
Comment (eguoli) [reply INSTALLATION.md:84]: Good point!

**Pull request merged**